processing/src/main/java/io/druid/query/lookup/LookupBean.java

```diff
@@ -19,32 +19,76 @@
 
 package io.druid.query.lookup;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Preconditions;
 
 //TODO merge this code to the same definition when pr/1576 is merged
```

Comment:
yeah, but I don't want to take care of that comment in this PR ... it is changing many things which are necessary for this PR already. will probably do this in a separate PR.